packages/devtools_app/lib/src/screens/memory/shared/heap/class_filter.dart

```diff
@@ -153,6 +154,41 @@ class ClassFilter {
     return filterType == ClassFilterType.except;
   }
 
+  static List<T> filter<T>({
```

Comment:
Can you add some documentation here? In particular, what are each of these parameters used for?